test/server/fixtures/field.html.ejs

```diff
@@ -26,7 +25,7 @@
 
       for (var i = 0; i < window.parent.frames.length; i++) {
         var frame = window.parent.frames[i].contentWindow || window.parent.frames[i];
-        inputs[frame.config().type] = frame.value();
+        if (frame !== undefined) inputs[frame.config().type] = frame.value();
```

Comment:
@chrissrogers : this is how I deal with the stale relays

Reply:
I think these frames shouldn't be ignored here, and the fact that they're causing issues indicates that they should be present.